src/main/java/co/org/osso/api/CustomerService.java

```diff
@@ -0,0 +1,68 @@
+package co.org.osso.api;
+
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class CustomerService {
+
+    public List<Customer> getCustomers() {
+        List<Customer> customersList = new ArrayList<>();
+        customersList.add(createCustomer(1111, "Test user 1", "test1@mail.com", "01-01-1901"));
+        customersList.add(createCustomer(2222, "Test user 2", "test2@mail.com", "02-02-1902"));
+        customersList.add(createCustomer(3333, "Test user 3", "test3@mail.com", "03-03-1903"));
+        return customersList;
+    }
+
+
+    public Customer createCustomer(Integer ID, String name, String email, String dob){
```

Comment:
I like that you created this method, but I think perhaps it should be a `private` method, as it's only being used inside this class? 
Can we rename `ID` to `id` please. I thought it was a constant (because it was all upper-case) when I read line 23.

Reply:
Done